tests/testProfileChangePassword.php

```diff
@@ -157,6 +159,43 @@ public function testSuccessfulPasswordChangeDuringWooAccountEdit() {
 		$this->assertEmpty( $errors->get_error_messages() );
 	}
 
+	/**
+	 * Test that the change password process handles escaped data.
+	 */
+	public function testThatPasswordIsUnescapedBeforeSending() {
+		$this->startHttpHalting();
+		self::$options->set( 'domain', 'example.auth0.com' );
+
+		$user   = $this->createUser();
+		$errors = new WP_Error();
+
+		// Store userinfo for a DB strategy user.
+		$this->storeAuth0Data( $user->ID, 'auth0' );
+
+		// Set a password with special characters.
+		$new_password        = uniqid() . '"' . uniqid() . "'" . uniqid() . '\\' . uniqid();
```

Comment:
I rather you use `uniqid()` plus all `(!@#$%^&*()_+=-;’|”:[]{}/.,<>?`~)` reported in the community to cover more symbols

Reply:
She mentioned those symbols but the only affected ones are listed here:

https://secure.php.net/manual/en/function.addslashes.php